wled00/FX.cpp

```diff
@@ -3385,58 +3391,57 @@ uint16_t mode_drip(void)
   struct virtualStrip {
     static void runStrip(uint16_t stripNr, Spark* drops) {
 
-      uint8_t numDrops = 1 + (SEGMENT.intensity >> 6); // 255>>6 = 3
+      uint8_t numDrops = map(SEGMENT.intensity, 0, 255, 1, 4);
 
-      float gravity = -0.0005 - (SEGMENT.speed/50000.0);
+      float gravity = -0.0005 - (SEGMENT.speed/25000.0); //increased max speed 50000->25000
```

Comment:
Did you test on short segments 10 pixels and/or long segments 300+ pixels?
May behave too fast if not tested.

Reply:
Yes, it is a range so all values possible